stylix/palette.nix

```
@@ -74,6 +80,26 @@ in {
         doesnt fix your monitor perfectly
       '';
     };
+    imageEditor = {
+      enable = lib.mkOption {
+        type = lib.types.bool;
+        default = false;
+        description = ''
+          Update Wallpaper by applying a lut filter to the image
+        '';
+      };
+      method = lib.mkOption {
+        type = with lib.types; functionTo (coercedTo package toString path);
+        default = config.lib.stylix.imageEditors.lutgen;
+        description = ''
+          A function to edit the image, takes one argument (the image)
+          and returns the resulting edited image
```

Simplify the function description:

```suggestion
          Edits the given 'stylix.image' argument.
```

Could you mention in the description where the edited image should be outputed to?